network/edge-services/how-to/subscribe-edge-services.mdx

```diff
@@ -0,0 +1,71 @@
+
+- A fixed number of Edge Services [pipelines](/network/edge-services/concepts/#pipeline). You can create pipelines for either Load Balancers, Object Storage buckets, or a mixture of both, with your subscription plan.
+- A certain amount of egress [cache](/network/edge-services/concepts/#cache) data (the quantity of data transferred from Edge Services' caches, not including the transfer from the origin bucket or Load Balancer).
+- A custom domain and SSL certificate (managed or custom) for each pipeline.
+
+If you create more pipelines than are included in your plan, or your pipelines' caches egress more data than is included, you will be charged additionally for this. The rates per pipeline/GB of data are indicated on the [pricing](https://www.scaleway.com/en/pricing/network/#edge-services) page.
+
+Subscriptions are pro-rata, meaning you can cancel your subscription or change your plan at any time, and you will be charged proportionally.
+
+Find out more about how Edge Service subscription plans and billing works on our [Understanding Edge Services pricing](/network/edge-services/reference-content/understanding-edge-pricing/) page.
+
+## How to subscribe to Edge Services
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+
+<Message type="tip">
+You can subscribe to Edge Services from the Object Storage or Load Balancer dashboards of the Scaleway console. However, the instructions below detail how to describe from the dedicated Edge Services section of the console. Whichever path you use, the same pricing plans and subscription mechanisms apply. Subscribing from the Object Storage or Load Balancer dashboards does **not** limit you to only creating pipelines for that product with your subscription.
+</Message>
+
+1. Click **Edge Services** in the **Network** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+
+2. Click **Subscribe to Edge Services**.
+
+    Information about available [pricing plans](https://www.scaleway.com/en/pricing/network/#edge-services) is displayed. All pricing plans have a fixed monthly price, and include a certain number of pipelines and a certain amount of cache data (data egressed from all your Edge Services pipelines' caches). Any extra pipelines or cache egress data are charged at the additional rates shown.
+
+3. Select a plan, and click **Subscribe to Edge Services**.
+
+    Your subscription is created, and you are returned to the Edge Services dashboard, on the **Pipelines** tab. All your Edge Services pipelines will display here, along with an indicator of your pipeline and cache consumption in relation to your monthly plan limits.
```

Comment:
We won't display the indicator at first (but it will be available quickly after the GA)